release/update_files.sh

```diff
@@ -25,3 +26,8 @@ find . -type f -name "*.ini" |
 # postgres_citus_versions: [('<new_pg_version>', '<new_branch1>')]
 find . -type f -name "*.ini" | 
     xargs sed -i "s@postgres_citus_versions: \[('[^,;]\+', '[^,;]\+')\]@postgres_citus_versions: \[('${new_pg_version}', '${new_branch1}')\]@g"
+# replace all .ini config files (except master.ini which requires use_enterprise to be off):
+# use_enterprise: <use_enterprise>
+# use_enterprise: <new_use_enterprise>
+find . -type f -name "*.ini" ! -name "master.ini" | 
```

Comment:
I think we can keep `master.ini` as well because we will not use this for hammerdb and not excluding anything seems simpler while doing the job